pex/bin/pex.py

```diff
@@ -121,6 +121,12 @@ def process_precedence(option, option_str, option_value, parser, builder):
   elif option_str in ('--no-manylinux', '--no-use-manylinux'):
     setattr(parser.values, option.dest, False)
     builder.no_use_manylinux()
+  elif option_str == '--transitive':
```

Comment:
Adding this to `process_precedence` seems like a stretch. The precedence setting in PEX deals with which to prefer, wheel or egg or sdist. The binary or not and manylinux options are fairly related but transitivity really isn't. Breaking out a separate callback for `process_intransitive` seems a bit more logical.